plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -0,0 +1,5 @@
+google = Google
+Bundle-Vendor = Google Inc.
+Bundle-Name = Wizards for Google Cloud Platform Apps
```

Comment:
Yes: http://spring.io/blog/2008/02/18/creating-osgi-bundles/
I thought the Bundle name was more human readable for display to people. If that's not true, what's the difference between Bundle-Symbolic_Name and Bundle-Name? Or what would you suggest here for the value of Bundle-Name?